specification/gedcom-3-structures-1-organization.md

```diff
@@ -400,19 +416,170 @@ A `SHARED_NOTE_RECORD` may contain a pointer to a `SOURCE_RECORD` and vice versa
 
 ```gedstruct
 n @XREF:SPLAC@ SPLAC <Text>                {1:1}  g71:record-SPLAC
-  +1 TYPE <Text>                           {0:1}  g71:SPLAC-TYPE
   +1 LANG <Language>                       {0:1}  g7:LANG
-  +1 TRAN <Text>                           {0:M}  g71:TRAN
-     +2 LANG <Language>                    {0:1}  g7:LANG
-  <<PLACE_DETAILS>>                        {1:1}
-  <<SHARED_PLACE_STRUCTURE>>               {0:M}  g71:SPLAC
+  +1 TYPE <Text>                           {1:1}  g71:SPLAC-TYPE 
+    +2 HREL <HIERARCHICAL_RELATIONSHIP>    {1:1}  g71:SPLAC-HREL
```

Comment:
The `_LOC` extension has `{0:M}` occurrences of hierarchical relationship (i.e., one per pointer that can be 0:M), to allow for multiple types of nesting, including ecclesiastical, geographical, governmental, historical, etc.